students/C_Farris/session04/Mailroom_2.py

```diff
@@ -188,14 +184,4 @@ def retry():
                 }
     prepare_to_write_to_disk()
 
-    main()
-
-
-
-
-
-
-
-
-
-
+    main()
```

Comment:
this is a bit odd that it's in this branch -- but no real changes, it may be from our attempt to clean things up.

not a biggie.